content/kb/troubleshooting-unable-to-mount-an-nfs-backup-target.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Troubleshooting: Unable to access an NFS backup target"
+author: James Lu
+draft: false
+date: 2022-05-27
+categories:
+  - "backup"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Prerequisite
+
+Longhorn only supports NFS v4.0, v4.1 and v4.2.
+
+### How to determine which NFS version is used
+
+Longhorn supports only NFS versions 4.0, 4.1, and 4.2. Users should ensure that both their backup server and client support one of the supported versions.
+
+* Server side:
+
+The NFS server can support multiple versions by the configuration, so using which version for each connection will actually depend on how the NFS share is mounted by the client.
```

Comment:
Remove this part. Just mentioning how to check is enough.

Reply:
ok, deleted